pkg/tecdsa/signing/message.go

```diff
@@ -71,3 +71,23 @@ func (trtm *tssRoundTwoMessage) SenderID() group.MemberIndex {
 func (trtm *tssRoundTwoMessage) Type() string {
 	return messageTypePrefix + "tss_round_two_message"
 }
+
+// tssRoundThreeMessage is a message payload that carries the sender's
+// TSS round three components.
+type tssRoundThreeMessage struct {
+	senderID group.MemberIndex
+
+	payload   []byte
```

Comment:
Same question about `payload` vs `broadcastPayload`. I see `pkg/tecdsa/dkg/message.go` is also inconsistent about it.

Reply:
Addressed by #3271